test/unit/jobs/cleanup-test.js

```diff
@@ -103,6 +110,39 @@ describe("Delete unused files", () => {
     }, 200);
   });
 
+  it("should delete financial data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
+      cleanupJob.financial.findByKey("risefinancial_realtime_HXXHCTR5AQFQ_-Kn5MaUE2cpYN-FCQSOU_1M_", (err, foundFinancial) => {
+        expect(Object.keys(foundFinancial.data).length).to.equal(0);
+
+        done();
+      })
+    }, 200);
+  });
+
+  it("should delete spreadsheet data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
```

Comment:
The tasks could be nested in each other's callbacks, or each callback could check to see if they're all done by checking some expected total.  Or if you change the callback style to a promise style you could use Promise.all() but I don't think it's worth changing styles at this point.

In each callback maybe something like:

``` js
if (!taskCallbackCount) {mainCallback();} else {taskCallbackCount--}
```